### PR TITLE
Consideration Review and History UXA

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#21796

*Description of changes:*
- Remove unnecessary formatting of consideration

- Minor MHR History UXA items - includes some api -> ui formatting

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1972-r9lcdwds.web.app

File: ppr-ui/src/views/mhrInformation/MhrHistory.vue [hunk 1]
```diff
@@ -190,6 +198,43 @@ const getHomeOwnerIcon = (partyType: HomeOwnerPartyTypes, isBusiness = false): s
...
 }
 
+/**
+ * Maps an API home location type to a UI home location type.
+ *
+ * @param {HomeLocationTypes} apiType - The API home location type to map.
+ * @returns {HomeLocationUiTypes} The corresponding UI home location type.
+ */
+function mapApiToUiLocationType(apiType: HomeLocationTypes): HomeLocationUiTypes {
+  console.log()
```
Comment (dimak1) [inline ppr-ui/src/views/mhrInformation/MhrHistory.vue]: Leftovers

**Review by dimak1 — APPROVED** [review]
LG!

File: ppr-ui/src/views/mhrInformation/MhrHistory.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/views/mhrInformation/MhrHistory.vue]: Thank you!

**Review by doug-lovett — APPROVED** [review]
Magnifique!

**Pull request merged**